crowbar_framework/app/models/service_object.rb

```diff
@@ -1543,9 +1561,10 @@ def run_remote_chef_client(node, command, logfile_name)
 
   private
 
-  def wait_for_chef_clients(node)
-    unless RemoteNode.chef_ready?(node, 1200)
-      STDERR.puts "Waiting for already running chef-clients on #{node} failed"
+  def wait_for_chef_clients(node_name)
+    @logger.debug("wait_for_chef_clients: Waiting for already running chef-clients on #{node_name}.")
+    unless RemoteNode.chef_ready?(node_name, 1200)
```

Comment:
@vuntz would https://github.com/crowbar/barclamp-crowbar/issues/1265 be good enough to resolve the logging issue here?